src/main/java/ru/rosatom/documentflow/dto/DocProcessCommentDto.java

```diff
@@ -0,0 +1,20 @@
+package ru.rosatom.documentflow.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+@Data
+@Schema(description = "Комментарий")
+public class DocProcessCommentDto {
+    @Schema(description = "Текст комментария")
+    private String textComment;
```

Comment:
Название переменной не очень хорошее. Старайся писать так как это звучало бы на английском, а не на русском с переводом отдельных слов. Приписка `Comment` тут не очень нужна т.к очевидно что текст пренадлежит комментарию 
```suggestion
    @Schema(description = "Текст комментария")
    private String content;
```